### ISSUE TITLE
Great program. Thanks.

### ISSUE THREAD

**Issue by rokester** [Dreams-builder/O365-Client#2]
Good program indeed. Is there likely to be **more functions**, e.g. Onedrive and Office in this program? *Thanks!*

**Comment by Micraow** [comment]
Yes. We have the plan, However, we are a group of students, the school is about to start, our ability is not particularly strong, so from now on, it is possible to achieve, but will be more difficult. We will try our best to meet your requirements. In the meantime, if you have certain development foundation, also welcome to join us.
Telegram:
https://t.me/joinchat/UaS-lBZ6XmizVuB6_9aGuw

Email:
pengbo@pengbo0708.onmicrosoft.com
xiaocao162020@xiaocao.onmicrosoft.com

**Comment by Micraow** [comment]
If you don't want to join us or keep in touch with us.
Here is another link

t.me/fuider

Thank you for giving us advise

**Comment by rokester** [comment]
You are welcome. I have joined the group. Thank you, too.

**Comment by c-jeremy** [comment]
> You are welcome. I have joined the group. Thank you, too.

@rokester Thank you to help us make the project better!
